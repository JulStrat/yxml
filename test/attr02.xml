<a a="a'b'&lt;&amp;&#x3c;&#x26;"/>
