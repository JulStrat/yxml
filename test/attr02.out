
elemstart a
attrstart a
attrval a'b'<&<&
attrend
elemend
ok
